src/OrbitClientGgp/ClientGgp.cpp

```diff
@@ -215,8 +217,14 @@ ErrorMessageOr<void> ClientGgp::LoadModuleAndSymbols() {
   // Load symbols for the module
   const std::string& module_path = main_module_->file_path();
   ORBIT_LOG("Looking for debug info file for %s", module_path);
-  OUTCOME_TRY(auto&& main_executable_debug_file,
+  OUTCOME_TRY(orbit_base::NotFoundOr<std::filesystem::path> && find_result,
               process_client_->FindDebugInfoFile(module_path, {}));
+  if (orbit_base::IsNotFound(find_result)) {
+    return ErrorMessage{
+        absl::StrFormat("Symbols not found: %s", orbit_base::GetNotFoundMessage(find_result))};
+  }
+  ORBIT_CHECK(std::holds_alternative<std::filesystem::path>(find_result));
```

Comment:
It's a bit weird to see these `std::holds_alternative` (also in other places in this change) because it shows an implementation detail of `NotFoundOr`.

Reply:
I now introduced a `GetFound` method (see next comment). As part of this, I decided a `IsFound(...)` method is not necessary, as its basically just `!IsNotFound(...)`. This made me realize that all the `ORBIT_CHECK(std::holds_alternative<...>(not_found_or))` come right after an if checking `IsNotFound()`, which is not really necessary, so I removed them.

Reply:
> This made me realize that all the `ORBIT_CHECK(std::holds_alternative<...>(not_found_or))` come right after an if checking `IsNotFound()`, which is not really necessary, so I removed them.

Yep!